barretenberg/cmake/arch.cmake

```diff
@@ -5,6 +5,7 @@ if(WASM)
     add_compile_options(-fno-exceptions -fno-slp-vectorize)
 endif()
 
-if(NOT WASM AND NOT APPLE AND NOT ARM)
-    add_compile_options(-march=skylake-avx512)
+if(NOT WASM AND NOT APPLE AND NOT ARM) 
+#    add_compile_options(-march=skylake-avx512) # Required for AVX-512 extensions
+    add_compile_options(-march=haswell)
```

Comment:
Upstream bb switched this to skylake at https://github.com/AztecProtocol/barretenberg/blob/master/cpp/cmake/arch.cmake#L9

Do you know the difference?

Reply:
Since I'm doing the work to use upstream bb in noir, it'd be great to keep these aligned! Can you test skylake on the Intel i7-7500U running Arch Linux you have access to?

Reply:
Confirmed working. I'll make the change.